index.src.html

```diff
@@ -619,7 +619,63 @@ <h3 id="data-minimization">
     Data Minimization
   </h3>
 
-  TODO
+  Minimization is a strategy that involves exposing as little information to
+  other communication partners as is required for a given operation to
+  complete. More specifically, it requires not providing access to more
+  information than was apparent in the user-mediated access or allowing the
+  user some control over which information exactly is provided.
+
+  For example, if the user has provided access to a given file, the object
+  representing that should not make it possible to obtain information about
+  that file's parent directory and its contents as that is clearly not what is
+  expected.
+
+  Basic fingerprinting guidance can be found here.
```

Comment:
Why fingerprinting in this place?  I see no connection (in the writeup).

Reply:
Reference from incorporating another document into this one.  Removed with https://github.com/w3cping/security-questionnaire/commit/ea8d5ac7be15a52e92108c4d6a65c1884c6065ca.